### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by YaninaGomezCucchiararo**

### CODE REVIEW

**Review by alexgbor — COMMENTED** [review]
Interesting way to make it adaptive! Feel free to check mine for a different approach.

File: staff/YaninaGomezCucchiararo/css-layout-simple-adaptive/style/main-desktop-css.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+html, body {
```
Comment (manuelbarzi) [inline staff/YaninaGomezCucchiararo/css-layout-simple-adaptive/style/main-desktop-css.css]: HINT too long file name... let's keep it simple. just ```desktop.css```is enough ,)

File: staff/YaninaGomezCucchiararo/css-layout-simple-adaptive/style/main-desktop-css.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    min-height:100% !important;
+    width: 100%;
+    height: 100%;
+}
+
+*{
```
Comment (manuelbarzi) [inline staff/YaninaGomezCucchiararo/css-layout-simple-adaptive/style/main-desktop-css.css]: TIP many of these css rules are repeated in both mobile and desktop. in that case, is convenient to concentrate them in a single file, like main.css. move them there.

File: staff/YaninaGomezCucchiararo/css-layout-simple-adaptive/style/main-desktop-css.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    padding: 0%;
+    margin: 0%;
+    color:white;
+    text-align: center;
+    border: 1px solid white;
+}
+
+header{
+    height: 20%;
+    width: 100%;
```
Comment (manuelbarzi) [inline staff/YaninaGomezCucchiararo/css-layout-simple-adaptive/style/main-desktop-css.css]: TIP is ```width: 100%```required?

File: staff/YaninaGomezCucchiararo/css-layout-simple-adaptive/style/main-desktop-css.css [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    background-color:black
+}
+
+main{
+    height: 60%;
+    width:100%;
+    background-color: black;
```
Comment (manuelbarzi) [inline staff/YaninaGomezCucchiararo/css-layout-simple-adaptive/style/main-desktop-css.css]: TIP same css rules for different selectors may be grouped

SEE Grouping Selectors https://www.w3schools.com/css/css_syntax.asp

File: staff/YaninaGomezCucchiararo/css-layout-simple-adaptive/style/main-mobile.css [hunk 5]
```diff
@@ -0,0 +1,55 @@
+
```
Comment (manuelbarzi) [inline staff/YaninaGomezCucchiararo/css-layout-simple-adaptive/style/main-mobile.css]: HINT main-mobile.css is a bit redundant name. let's keep it simple, just mobile.css is enough. rename it.

File: staff/YaninaGomezCucchiararo/cv/styles/mainmobile.css [hunk 6]
```diff
@@ -0,0 +1,72 @@
+html, body {
```
Comment (manuelbarzi) [inline staff/YaninaGomezCucchiararo/cv/styles/mainmobile.css]: TIP mobile.css instead of mainmobile.css? mainmobile is too long and difficult to read

**Comment by manuelbarzi** [comment]
PLEASE review comments and apply changes! otherwise PR will grow too much... need to merge soon ,)

File: staff/YaninaGomezCucchiararo/cv/styles/maindesktop.css [hunk 7]
```diff
@@ -0,0 +1,88 @@
+html, body {
```
Comment (manuelbarzi) [inline staff/YaninaGomezCucchiararo/cv/styles/maindesktop.css]: RENAME maindesktop.css to desktop.css

File: staff/YaninaGomezCucchiararo/js/challenges/count-chars/scripts/countChars.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
+function countChars(str, func) {
```
Comment (agandia9) [inline staff/YaninaGomezCucchiararo/js/challenges/count-chars/scripts/countChars.js]: use strict in all js files!...

File: staff/YaninaGomezCucchiararo/js/challenges/scripts/count-chars.test.js [hunk 9]
```diff
@@ -0,0 +1,28 @@
+var count = countChars('Hello world');
```
Comment (agandia9) [inline staff/YaninaGomezCucchiararo/js/challenges/scripts/count-chars.test.js]: use strict!

File: staff/YaninaGomezCucchiararo/js/challenges/scripts/count-words.js [hunk 10]
```diff
@@ -0,0 +1,6 @@
+function countWords(str){
```
Comment (agandia9) [inline staff/YaninaGomezCucchiararo/js/challenges/scripts/count-words.js:3]: use strict!

File: staff/YaninaGomezCucchiararo/js/challenges/scripts/countChars.js [hunk 11]
```diff
@@ -0,0 +1,28 @@
+function countChars(str, func) {
```
Comment (agandia9) [inline staff/YaninaGomezCucchiararo/js/challenges/scripts/countChars.js]: use strict!

File: staff/YaninaGomezCucchiararo/js/challenges/scripts/to-camel-case.test.js [hunk 12]
```diff
@@ -0,0 +1,3 @@
+var res= toCamelCase('Hello My World');
```
Comment (agandia9) [inline staff/YaninaGomezCucchiararo/js/challenges/scripts/to-camel-case.test.js]: use strict!

File: staff/YaninaGomezCucchiararo/js/challenges/scripts/to-random-case.js [hunk 13]
```diff
@@ -0,0 +1,20 @@
+function toRandomCase(str) {
```
Comment (agandia9) [inline staff/YaninaGomezCucchiararo/js/challenges/scripts/to-random-case.js:3]: **use strict**!

File: staff/YaninaGomezCucchiararo/js/challenges/scripts/to-roman-number.js [hunk 14]
```diff
@@ -0,0 +1,9 @@
+function toRomanNumber(num){
```
Comment (agandia9) [inline staff/YaninaGomezCucchiararo/js/challenges/scripts/to-roman-number.js:3]: use strict!

File: staff/YaninaGomezCucchiararo/js/challenges/scripts/to-roman-number.test.js [hunk 15]
```diff
@@ -0,0 +1,10 @@
+var res= toRomanNumber("m");
```
Comment (agandia9) [inline staff/YaninaGomezCucchiararo/js/challenges/scripts/to-roman-number.test.js]: use strict!

**Comment by YaninaGomezCucchiararo** [comment]
Done! =)

File: staff/YaninaGomezCucchiararo/js/challenges/scripts/camel-case.js [hunk 16]
```diff
@@ -0,0 +1,12 @@
+'use strict';
+function toCamelCase(str){
+    if(typeof str === "String"){
```
Comment (Asix94) [inline staff/YaninaGomezCucchiararo/js/challenges/scripts/camel-case.js]: is "string" no "String", the first s in lowerCase

File: staff/YaninaGomezCucchiararo/js/challenges/scripts/reduce.js [hunk 17]
```diff
@@ -0,0 +1,81 @@
...
+var a = [{ name: 'jeans', price: 10.5 }, { name: 't-shirt', price: 5.99 }, { name: 'socks', price: 19.99 }];
+
+reduce(a, function(accum, v) {
+if (v.price > 10)
+return accum + v.price;
+
+return accum;
+}, 0); // -> 30.49*/
+
+'use strict'
```
Comment (agandia9) [inline staff/YaninaGomezCucchiararo/js/challenges/scripts/reduce.js]: use strict goes to top of document...

File: staff/YaninaGomezCucchiararo/js/challenges/scripts/reduce.js [hunk 18]
```diff
@@ -0,0 +1,81 @@
...
+        accum=accum+a[i];
+        if( a[i] === v){
+            break;
+        } 
+    }
+    
+
+}
+*/
+'use strict';
```
Comment (agandia9) [inline staff/YaninaGomezCucchiararo/js/challenges/scripts/reduce.js]: ....

File: staff/YaninaGomezCucchiararo/js/challenges/scripts/reduce.js [hunk 19]
```diff
@@ -0,0 +1,81 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (agandia9) [inline staff/YaninaGomezCucchiararo/js/challenges/scripts/reduce.js]: WOT.

**Pull request merged**